README.md

```diff
@@ -121,9 +124,8 @@ rubocop
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

![image](https://github.com/dieum512/OOP/assets/93039359/2e7bf864-0cd0-4054-b396-b75c26b85d78)

- [ ] First of all, I would apologize on behalf of the previous reviewer but I assure you if you make these changes then your app will work as expected and according to project requirements 👍  It would be nice if you make some changes in your app because your app is broken right now 😨 Kindly follow the below comments to fix this issue to make your app work as expected 👍 😊